kroki/plugin.py

```diff
@@ -21,7 +22,7 @@
 
 class KrokiPlugin(BasePlugin):
     config_scheme = (
-        ('ServerURL', config.config_options.Type(str, default='https://kroki.io')),
+        ('ServerURL', config.config_options.Type(str, default=os.getenv('KROKI_SERVER','https://kroki.io'))),
```

Comment:
Could you please change the OS-Var to KROKI_SERVER_URL to be more consistent with the config file